app/views/find/search/locations/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<% content_for :page_title, title_with_error_prefix(t("find.homepage.title"), @courses_by_location_or_training_provider_form.errors.present?) %>
```

Comment:
Can we remove duplicate page in the other location?

Reply:
Oh nice one, I thought I forgot something...